doc/api/vm.md

```diff
@@ -866,6 +866,36 @@ local scope, so the value `localVar` is changed. In this way
 `vm.runInThisContext()` is much like an [indirect `eval()` call][], e.g.
 `(0,eval)('code')`.
 
+## vm.compileFunction
+<!-- YAML
+added: REPLACEME
+-->
+* `code` {string} The JavaScript code to compile and run.
+* `params` {Array|undefined} An array of string containing all paramters for
```

Comment:
1. `An array of string` -> `An array of strings`.
2. `paramters` -> `parameters`.
3. An array of strings is usually expressed as `string[]`.
4. IIRC, we usually just wrap a parameter in `[]` and not enumerate `undefined`. See, for example, `vm.runInNewContext(code[, sandbox][, options])` section and `sandbox` description.